examples/data_visualization_p300.py

```diff
@@ -0,0 +1,210 @@
+import warnings
+from pathlib import Path
+
+import matplotlib
+import mne
+import numpy as np
+import seaborn as sns
+from matplotlib import pyplot as plt
+
+from moabb.paradigms import P300
+
+
```

Comment:
Add a warning that this scripts need to work on all P300 datasets, that is several hundred of Gb.